packages/checkout/widgets-lib/src/widgets/sale/SaleWidget.tsx

```diff
@@ -183,6 +187,10 @@ export default function SaleWidget(props: SaleWidgetProps) {
               subheading={viewState.view.data?.subheading}
             />
           )}
+          <CreditCardWarningDrawer
+            visible={showCreditCardWarning}
```

Comment:
I was wondering how does this get triggered now? After the move I can't see how it links up from the payment methods 🤔

Reply:
accidentally removed in the latest `Clean up` commit, reverted it back